### ISSUE TITLE
Replication Metric - column lag_b does not exists

### ISSUE THREAD

**Issue by cleitondomazak** [cybertec-postgresql/pgwatch2#101]
I'm run the pw2 with docker image, and I use my influxdb and Grafana installation also. And in the replication query does not return any row. I figure out it is happing because don't exist the column ```lag_b``` in influxdb.

The metric query(from pw2 repository):
```sql
SELECT mean("lag_b") FROM "replication" WHERE $timeFilter GROUP BY time($interval), "dbname" fill(none)
```

The table ```replication``` have the following schema:
**time, application_name, dbname, replay_lag_b, write_lag_b**

![image](https://user-images.githubusercontent.com/8502873/44260381-1fb40980-a20c-11e8-9912-15ffd56ea25d.png)

**Versions**
**PW2:** "pgwatch2_git_version": "9eb5ac699df9873d2138f94dc35e1ba509dd82a6"
**Influxdb(external):** 1.3.1
**PostgreSQL:** 10.4.2

**Comment by kmoppel** [comment]
Hi @cleitondomazak ! I think you need to update to the most recent version of pgwatch2 image - or in your case you need to reload the metric definitions and the "Replication" dashboard JSON as you're using a custom setup. This got changed only quite recently.

**Comment by cleitondomazak** [comment]
I downloaded the last docker image, but the same problem happens. But maybe I understand the real "problem", correct me if I am wrong, but the column _lag_b_ does not exists, and was replaced for the 2 new ones, and is needed to change the Grafanas query of Replication Dashboard. But in v4 dashboards, this query is the old one yet. Is correct? 

https://github.com/cybertec-postgresql/pgwatch2/blob/32534eee2087176338cc9ac1629eff757d04ac8a/grafana_dashboards/v4/replication/dashboard.json#L72
